pkg/sdk/parsers_test.go

```diff
@@ -61,11 +61,41 @@ func TestParseCommaSeparatedStringArray(t *testing.T) {
 			Result: []string{"one", "two", "three"},
 		},
 		{
-			Name:       "list without brackets - with quotes",
+			Name:       "list without brackets - with single quotes",
 			Value:      "'one','two','three'",
 			TrimQuotes: true,
 			Result:     []string{"one", "two", "three"},
 		},
+		{
+			Name:       "list without brackets - with double quotes",
+			Value:      `"one","two","three"`,
+			TrimQuotes: true,
+			Result:     []string{"one", "two", "three"},
+		},
+		{
+			Name:       "list with brackets - with double quotes",
+			Value:      `"one","two","three"`,
```

Comment:
This one should have brackets (same in the next test case)

Reply:
I will change the name of the test in the follow-up PR